### DISCUSSION TITLE
Errr Syntax error in input(1).

### DISCUSSION THREAD

**Discussion by kothapraveen** [swig/swig#2979] [General]
Hi @anatol 
#define STD(name)		std::name

const STD (wstring)DATE_NULL_STRINGW (L"00000000");
const STD(string)	DATE_NULL_STRING  ("00000000");

SWIG Version 3.0.9
we have included header file, which contains above lines in interface file, while compiling .i file facing below error

Error		Syntax error in input(1).

Interface file has below lines of code
################################
namespace std {
%apply const string & {string &};
};

%apply const std::wstring & {std::wstring &};
##############################################

**Comment by ojwb** [comment]
You reported the same thing as an issue too (#2980) - please don't do that, it just wastes everyone's time.  Closing.

Also, SWIG 3.0.9 is over 8 years old.
